index.js

```diff
@@ -275,6 +275,9 @@ function onReady () {
             case consts.eventNames.windowCmdUnmaximize:
             elements[json.targetID].unmaximize()
             break;
+            case consts.eventNames.windowCmdWebContentsExecuteJavascript:
+            elements[json.targetID].webContents.executeJavaScript(json.code).then((result) => client.write(json.targetID, consts.eventNames.webContentsExecuteJavaScript, {result}));
```

Comment:
I don't really see the point of using `result` here since the response body is not processed in GO. I would not return any body.